cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1233,7 +1236,7 @@
     <name>router.bind.address</name>
     <value>0.0.0.0</value>
     <description>
-      Router server address
+      CDAP Router bind address
```

Comment:
we should be consistent across the main CDAP processes (kafka, router, master, auth, ui).  We have ``Kafka service bind address`` vs ``CDAP Router bind address`` vs ``CDAP Authentication Server bind address``.  Compare ``CDAP`` or not, ``service``/``server`` or not